src/main/resources/assets/elevatorid/lang/pl_pl.json

```diff
@@ -0,0 +1,39 @@
+{
+  "block.elevatorid.elevator_white": "Biała winda",
+  "block.elevatorid.elevator_black": "Czarna winda",
+  "block.elevatorid.elevator_blue": "Niebieska winda",
+  "block.elevatorid.elevator_brown": "Brązowa winda",
+  "block.elevatorid.elevator_cyan": "Błękitna winda",
+  "block.elevatorid.elevator_gray": "Szara winda",
+  "block.elevatorid.elevator_green": "Zielona winda",
+  "block.elevatorid.elevator_light_blue": "Jasnoniebieska winda",
+  "block.elevatorid.elevator_lime": "Jasnozielona winda",
+  "block.elevatorid.elevator_magenta": "Karmazynowa winda",
+  "block.elevatorid.elevator_orange": "Pomarańczowa winda",
+  "block.elevatorid.elevator_pink": "Różowa winda",
+  "block.elevatorid.elevator_purple": "Fioletowa winda",
+  "block.elevatorid.elevator_red": "Czerwona winda",
+  "block.elevatorid.elevator_light_gray": "Jasnoszara winda",
+  "block.elevatorid.elevator_yellow": "Żółta winda",
+
+  "block.elevatorid.dir_elevator_white": "Kierunkowa biała winda",
+  "block.elevatorid.dir_elevator_black": "Kierunkowa czarna winda",
+  "block.elevatorid.dir_elevator_blue": "Kierunkowa niebieska winda",
+  "block.elevatorid.dir_elevator_brown": "Kierunkowa brązowa winda",
+  "block.elevatorid.dir_elevator_cyan": "Kierunkowa błękitna winda",
+  "block.elevatorid.dir_elevator_gray": "Kierunkowa szara winda",
+  "block.elevatorid.dir_elevator_green": "Kierunkowa zielona winda",
+  "block.elevatorid.dir_elevator_light_blue": "Kierunkowa jasnoniebieska winda",
+  "block.elevatorid.dir_elevator_lime": "Kierunkowa jasnozielona winda",
+  "block.elevatorid.dir_elevator_magenta": "Kierunkowa karmazynowa winda",
+  "block.elevatorid.dir_elevator_orange": "Kierunkowa pomarańczowa winda",
+  "block.elevatorid.dir_elevator_pink": "Kierunkowa różowa winda",
+  "block.elevatorid.dir_elevator_purple": "Kierunkowa fioletowa winda",
+  "block.elevatorid.dir_elevator_red": "Kierunkowa czerwona winda",
+  "block.elevatorid.dir_elevator_light_gray": "Kierunkowa jasnoszara winda",
+  "block.elevatorid.dir_elevator_yellow": "Kierunkowa żółta winda",
+
+  "elevatorid.subtitle.teleport": "Używana winda",
+
+  "itemGroup.tabElevators": "Elevators"
```

Comment:
Missing this one ?

Reply:
Items group = mod name. I think that should be like this.